source/cli/appservices-push.txt

```diff
@@ -44,23 +44,23 @@ Options
    * - --local
      - string
      - false
-     - Specify the local filepath of an app to be imported
+     - Specify the local filepath of an app to be pushed
```

Comment:
```suggestion
     - Specify the local filepath of an app to be imported
```
Just double checking, were these wording changes from push to import made in the new set or did you make them? We normally accept the description as they come from engineering

Reply:
These were changed from the generated doc, I did not make them manually!